internal/cache/controller_obj_cache.go

```diff
@@ -113,6 +115,12 @@ func (c *AviObjCache) AviRefreshObjectCache(client []*clients.AviClient, cloud s
 		defer wg.Done()
 		c.PopulateL4PolicySetToCache(client[6], cloud)
 	}()
+	go func() {
+		if len(client) > 8 {
```

Comment:
This `if loop` is not required. We already creating `avi client of length 9 at https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/master/internal/cache/avi_ctrl_clients.go#L61

Reply:
I should have elaborated the comment. Intention was to check the length of aviclient in code. If it is not as per our requirement, increase over there.

Reply:
Reverted the check